bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -256,7 +256,7 @@ public String getKey() {
     @Override
     public void storeKey(@Nullable String key) {
         if (!getKey().equals(key)) {
-            logger.debug("store new key");
+            logger.info("Store {} as access Key in the thing configuration", key);
```

Comment:
If the key is sensitive information, please do not log it at all. Maybe log the number of characters.

Reply:
We need to give a chance to the user to get the value of the access key. The user will need it to set up his thing in a config file.

Please note that this log will appear only when the access key is not yet defined in the thing configuration, that means when first setting up the thing. As soon as the access key is set in the thing configuration, this code will no more be reached and so the access key will no more be logged.

This is done like that in few other bindings, when the key is a required configuration setting which is provided by the device at paring time (hue or freebox bindings for example).

One alternative would be to not log the access key as this place and make the access key configuration setting always visible in Paper UI. But in this case, the access key is ALWAYS accessible in Paper UI. I think it is less secure than the solution I propose.

Reply:
Agreed. We can leave it like that. I would prefer a different strategy, e.g. a console command that can print the current access key from memory. That way we could prevent logging at all. But that is out of scope of this PR.

Reply:
I agree with you, the console command is a good idea. I will add this feature.